install/upgrade/versions/1.3.3.sh

```diff
@@ -16,4 +16,92 @@ if [ -e "/etc/nginx/nginx.conf" ]; then
+            echo '        please review the /etc/exim4/exim4.conf.template'
+            echo '        file and resolve any conflicts.'
+            disable_smarthost=false
+        fi
+
+        # Add smarthost macros to exim config
+        insert=$(cat << EOL
+        SMARTHOST_FILE = \${if exists{/etc/exim4/domains/\${sender_address_domain}/smarthost.conf}{/etc/exim4/domains/\$sender_address_domain/smarthost.conf}{/etc/exim4/smarthost.conf}}\n\
+        SMARTHOST_HOST=\${lookup{host}lsearch{SMARTHOST_FILE}}\n\
+        SMARTHOST_PORT=\${lookup{port}lsearch{SMARTHOST_FILE}}\n\
+        SMARTHOST_USER=\${lookup{user}lsearch{SMARTHOST_FILE}}\n\
+        SMARTHOST_PASS=\${lookup{pass}lsearch{SMARTHOST_FILE}}\n
+        EOL
+        )
+
+        if [ "$disable_smarthost" = true ]; then
+            insert=$(sed 's/^/#/g; s/\\n/\\n#/g; s/.$//' <<< $insert)
+        fi
+
+        line=$(expr $(sed -n '/ACL CONFIGURATION/=' /etc/exim4/exim4.conf.template) - 1)
+        sed -i "${line}i $insert" /etc/exim4/exim4.conf.template
+
+        # Add smarthost authenticator
+        insert=$(cat << EOL
+        smarthost_login:\n\
+          driver = plaintext\n\
+          public_name = LOGIN\n\
+          hide client_send = : SMARTHOST_USER : SMARTHOST_PASS\n
+        EOL
+        )
+
+        if [ "$disable_smarthost" = true ]; then
+            insert=$(sed 's/^/#/g; s/\\n/\\n#/g; s/.$//' <<< $insert)
+        fi
+
+        line=$(expr $(sed -n '/begin authenticators/=' /etc/exim4/exim4.conf.template) + 2)
+        sed -i "${line}i $insert" /etc/exim4/exim4.conf.template
+
+        # Add smarthost router
+        insert=$(cat << EOL
+        send_via_smarthost:\n\
+          driver = manualroute\n\
+          address_data = SMARTHOST_HOST:SMARTHOST_PORT\n\
+          domains = !+local_domains\n\
+          require_files = SMARTHOST_FILE : !/etc/exim4/domains/\$sender_address_domain/no_smarthost\n\
+          transport = smarthost_smtp\n\
+          route_list = * \${extract{1}{:}{\$address_data}}::\${extract{2}{:}{\$address_data}}\n\
+          no_more\n\
+          no_verify\n
+        EOL
+        )
+
+        if [ "$disable_smarthost" = true ]; then
+            insert=$(sed 's/^/#/g; s/\\n/\\n#/g; s/.$//' <<< $insert)
+        fi
+
+        line=$(expr $(sed -n '/begin routers/=' /etc/exim4/exim4.conf.template) + 2)
+        sed -i "${line}i $insert" /etc/exim4/exim4.conf.template
+
+        # Add smarthost transport
+        insert=$(cat << EOL
+        smarthost_smtp:\n\
+          driver = smtp\n\
+          hosts_require_auth = \$host_address\n\
+          hosts_require_tls = \$host_address\n
+        EOL
+        )
+
+        if [ "$disable_smarthost" = true ]; then
+            insert=$(sed 's/^/#/g; s/\\n/\\n#/g; s/.$//' <<< $insert)
+        fi
+
+        line=$(expr $(sed -n '/begin transports/=' /etc/exim4/exim4.conf.template) + 2)
+        sed -i "${line}i $insert" /etc/exim4/exim4.conf.template
+    fi
```

Comment:
Whole script didn't get called during the upgrade we should rename everything to 1.4.0.sh